gammapy/datasets/simulate.py

```diff
@@ -206,7 +206,9 @@ def _sample_coord_time_energy(self, dataset, model):
 
         coords = npred.sample_coord(n_events=n_events, random_state=self.random_state)
 
-        coords["time"] = Time(coords["time"], format="mjd", scale="tt")
+        coords["time"] = Time(
+            coords["time"], format="mjd", scale=dataset.gti.time_ref.scale
```

Comment:
You could also use the `time_ref` on the `npred` time axis no? This should be the closest to the time scales of events sampled from `npred`.

Reply:
thanks @registerrier . Just to understand, how do you access to the `time_ref` of the `npred`? The `time` axis of `npred` seems to have only a list of times without any specification on `scales` and `format`..

Reply:
There should be a `reference_time` attribute in `TimeMapAxis`:
https://docs.gammapy.org/dev/api/gammapy.maps.TimeMapAxis.html#gammapy.maps.TimeMapAxis.reference_time

So accessing it via `npred.geom.axes["time"].reference_time` should be possible, no?

Reply:
Actually, it does not have such an attribute. Because `npred` (for energy-dependent temporal models) has a time axis created by `MapAxis` and not `TimeMapAxis`.